torch_xla/csrc/elementwise.cpp

```diff
@@ -141,10 +141,16 @@ xla::XlaOp BuildSoftshrink(xla::XlaOp input, const at::Scalar& lambda) {
 }
 
 xla::XlaOp BuildShrinkBackward(xla::XlaOp grad_output, xla::XlaOp input,
-                               const at::Scalar& lambda) {
+                               xla::XlaOp lambda) {
```

Comment:
nit: we may want to do `MaybeCast` similar to https://github.com/pytorch/xla/pull/3999/commits/4ccfc24a86f1ac1f771a466ef3513b0ce31104ad

Reply:
done